apps/docs/components/blocks/SelectDropdown.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: DefaultLayout
+hideBreadcrumbs: true
+description: This component is an alternative option for SfSelect component. It doesn't use native select tag to allow more styling customizations.
+
```

Comment:
Hide ToC.

Reply:
Do we have a decision to hide it always for blocks? In case of this block it does not matter if it's opened or closed (does not affect the visuals of the block).

Reply:
AFAIK we hide ToC for all blocks.

Reply:
then we need to fix some blocks on current dev 😄 
Added it here: ada92217d077e6ad1f3f531bcf1aadb5d4cb9551